hassio/src/addon-view/hassio-addon-info.js

```diff
@@ -381,6 +382,16 @@ class HassioAddonInfo extends EventsMixin(PolymerElement) {
                 ><mwc-button>Open web UI</mwc-button></a
               >
             </template>
+            <template
```

Comment:
I know you're busy, so I'll let your use of Polymer slide 😜

Reply:
I would love to convert all of hass.io to Lit. There are some other design patterns in there too that could be improved.